### DISCUSSION TITLE
Multi Agent for Vehicle

### DISCUSSION THREAD

**Discussion by Ghazimuhammad** [LucasAlegre/sumo-rl#219] [Q&A]
Can i use sumo-rl for multi agent with vehicle? spesifically for 1 TL agent and 5 AV agent? and, can i customized the apply_action?

**Comment by LucasAlegre** [comment]
Control of vehicles is not implemented in sumo-rl. However, if you want to implement such an environment, I believe forking sumo-rl and working on top of it to implement all the necessary changes might be easier.
